### ISSUE TITLE
Issue when array is null

### ISSUE THREAD

**Issue by JCtst** [fastify/fast-json-stringify#138]
Have issue, when array is null. 

```
'use strict';

const fastJsonStringify = require('fast-json-stringify');

const f = fastJsonStringify({
  type: 'object',
  properties: {
    data: {
      type: ['integer', 'null']
    }
  }
});

console.log(f({data:null}))
```

with result:

      throw er; // Unhandled 'error' event
      ^

     TypeError: Cannot read property 'length' of null

**Comment by mcollina** [comment]
This has a PR in progress that has stalled. Would you like to help bringing it over the finishing line: https://github.com/fastify/fast-json-stringify/pull/134?

**Comment by cemremengu** [comment]
This should be fix now that #134 has landed
